src/Microsoft.Fast.Components.FluentUI/Components/InputFile/FluentInputFile.razor

```diff
@@ -0,0 +1,36 @@
+﻿@namespace Microsoft.Fast.Components.FluentUI
+@using Microsoft.AspNetCore.Components.Forms
+@inherits FluentComponentBase
+
+<div class="@ClassValue"
+     style="@StyleValue"
+     drop-files=@DropOver
+@ondragenter="@(e => DropOver = true)"
+@ondragenter:stopPropagation
+@ondragover="@(e => DropOver = true)"
+@ondragover:stopPropagation
+@ondragleave="@(e => DropOver = false)"
+@ondragleave:stopPropagation
+@ondragend="@(e => DropOver = false)">
+
+    <div class="inputfile-content" style="z-index: @(DropOver ? null : 999)">
```

Comment:
Could you define a value in the `ZIndex` class to centralize (and allow future adaptations)?